### PR TITLE
Support v4 API

### PR DESCRIPTION
**Pull request by m1guelpf**
See #591

### CODE REVIEW

**Comment by acrobat** [comment]
Should we add a `configure` method or set the correct api version automatically in the  `execute` method?

**Comment by m1guelpf** [comment]
@acrobat @Nyholm Done!

File: lib/Github/Api/GraphQL.php [hunk 1]
```diff
@@ -2,23 +2,27 @@
 
 namespace Github\Api;
 
+use AcceptHeaderTrait;
```
Comment (acrobat) [inline lib/Github/Api/GraphQL.php]: Remove this use (duplicate use on line 17)

File: lib/Github/Api/GraphQL.php [hunk 2]
```diff
@@ -5,20 +5,22 @@
...
- * Part of the Github API Early-Access Program
+ * Part of the Github v4 API
  *
- * @link   https://developer.github.com/early-access/graphql/
+ * @link   https://developer.github.com/v4/
  * @author Miguel Piedrafita <soy@miguelpiedrafita.com>
  */
 class GraphQL extends AbstractApi
 {
+    use AcceptHeaderTrait;
```
Comment (GrahamCampbell) [inline lib/Github/Api/GraphQL.php]: missing newline after the semicolon

Comment (m1guelpf) [inline lib/Github/Api/GraphQL.php]: Done!

**Comment by m1guelpf** [comment]
@acrobat @Nyholm Updates?

**Comment by acrobat** [comment]
Looks good to me, thanks @m1guelpf!

**Pull request merged**